assignments/assignment01.rb

```diff
@@ -38,13 +10,21 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  # your implementation here
+    num_letters = s.length	#Get number of characters in string
+	s.downcase!		#Turn all letters to lowercase
+	s.capitalize!		#Capitalize the first word in the string
+	(1...num_letters).each do |n|		#Loop to capitalize any letter after a space
```

Comment:
there are simpler ways to approach this problem. Think about splitting the string into separate words, then capitalize each word, then join them back together again.